cdc_rsync_server/cdc_rsync_server.cc

```diff
@@ -152,7 +257,7 @@ PathFilter::Rule::Type ToInternalType(
 
 }  // namespace
 
-CdcRsyncServer::CdcRsyncServer() = default;
+CdcRsyncServer::CdcRsyncServer() {}
```

Comment:
Why this change?

The `= default;` style has no disadvantages and should be the default because it keeps the default constructor trivial if possible. (Not really relevant here though, because the class has members with non-trivial default constructors, so this constructor will also not be trivial. So feel free to ignore.)

Reply:
I have no memories of this change. I probably added a constructor arg at some point and removed it. Reverted.